app/MainHie.hs

```diff
@@ -98,6 +103,7 @@ main = do
 
 run :: GlobalOpts -> IO ()
 run opts = do
+  hSetBuffering stderr LineBuffering
```

Comment:
I don't remember why this is here. Probably not needed.

Reply:
I added this because multiple threads are writing to stderr and the output becomes interleaved at the character level.